src/User32.Desktop/User32+DISPLAYCONFIG_PATH_SOURCE_INFO.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="DISPLAYCONFIG_PATH_SOURCE_INFO"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        public struct DISPLAYCONFIG_PATH_SOURCE_INFO
+        {
+            public LUID adapterId;
+
+            public uint id;
+
+            public DISPLAYCONFIG_MODE_GROUP_UNION modeGroup;
+
+            public uint statusFlags;
```

Comment:
Bonus points for defining the Flags enum for this and any other structs you define.